src/renderer/App.tsx

```diff
@@ -73,140 +51,89 @@ declare global {
   }
 }
 
-function TooltipNavItem({
```

Comment:
does this mean there are no defaults anymore?

Reply:
I couldn't exactly find any reason for defaults to be specified. If a nav item was defined, I'd assume that the author of the code would need to fill in all of the properties correctly, right? Why define defaults if they're never (or rather shouldn't) be used?